config/config.hocon.sample

```diff
@@ -0,0 +1,51 @@
+{
+  # Human-readable storage target name, used only for logging
+  "name": "Acme Ltd. Snowplow Postgres"
+  # Machine-readable unique identifier
+  "id": "5c5e4353-4eeb-43da-98f8-2de6dc7fa947"
+
+  "input": {
+    # Enable the Kinesis source
+    "type": "Kinesis"
+    # Name of the application which the KCL daemon should assume
+    "appName": "acme-postgres-loader"
+    # Name of the Kinesis stream to read from
+    "streamName": "enriched-events"
+    # AWS region in which the Kinesis stream resides.
+    "region": "eu-central-1"
+    # Either TRIM_HORIZON or LATEST
+    "initialPosition": "TRIM_HORIZON"
+
+    # Optional, set the  polling mode for retrieving records. Default is FanOut
+    # "retrievalMode": "FanOut"
+    # "retrievalMode": {
+    #   "type": "Polling"
+    #   "maxRecords": 1000
+    # }
+
+    # Optional override to disable cloudwatch metrics
+    # "disableCloudWatch": true
```

Comment:
I wonder if this should be in a metrics section like for the [new s3 loader](https://github.com/snowplow/snowplow-s3-loader/blob/1165b7edd57bf6c207c2c60d9ca99a7bc793eaf7/config/config.hocon.sample#L70)

Reply:
I think yes and should be named the same. It's exactly the same setting.

Reply:
I actually copied `disableCloudWatch` from [stream enrich config](https://github.com/snowplow/enrich/blob/a47072f352d9b70f08fbc31b4d5fd78dbbf1e811/config/config.hocon.sample#L78) and [the elasticsearch loader config](https://github.com/snowplow/snowplow-elasticsearch-loader/blob/3ed0746c56f9be880259edc9971b87712d5830e7/examples/config.hocon.sample#L92) and [the old s3 loader config](https://github.com/snowplow/snowplow-s3-loader/blob/4157b9c768ec58d773c905cdd8d898ebccbf7c2c/examples/config.hocon.sample#L67)

Two of those three links will soon be obsolete anyway.  And the elasticsearch loader needs a redesign.  So I'm happy to say that was the old way of doing things, and going forwards we copy the new s3 loader.